aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md

```diff
@@ -25,37 +25,37 @@ The MVC Music Store is a tutorial application that introduces and explains step-
 
 The application we'll be building is a simple music store. There are three main parts to the application: shopping, checkout, and administration.
 
-![](mvc-music-store-part-1/_static/image1.jpg)
+![Screenshot of the ASP.Net Music Store overview menu, with options for selecting a genre or from the top picks selections.](mvc-music-store-part-1/_static/image1.jpg)
 
 Visitors can browse Albums by Genre:
 
-![](mvc-music-store-part-1/_static/image2.jpg)
+![Screenshot of the ASP.NET music store genre albums selections menu that shows the collection of albums in a given genre.](mvc-music-store-part-1/_static/image2.jpg)
 
 They can view a single album and add it to their cart:
 
-![](mvc-music-store-part-1/_static/image3.jpg)
+![Screenshot of the album selection window, showing the album's name, genre, artist, and price, with an option to add to cart.](mvc-music-store-part-1/_static/image3.jpg)
 
 They can review their cart, removing any items they no longer want:
 
-![](mvc-music-store-part-1/_static/image4.jpg)
+![Screenshot of the 'Review your cart' menu, with total price information and options to edit your cart or checkout. ](mvc-music-store-part-1/_static/image4.jpg)
```

Comment:
Remove quotes from "Review your cart". We should actually clarify this with Derek!